# HG changeset patch
# User Michael Palimaka <kensington@gentoo.org>
# Date 1339067401 -36000
# Node ID 095d2f074f89ba34bec0c8d96e54a048ce24ff93
# Parent  0cbee16e4e3dc60fcf192035b60f44ae4cd6ca40
Fix build with GCC 4.7, by Tom Callaway <spot@fedoraproject.org>.

diff -r 0cbee16e4e3dc60fcf192035b60f44ae4cd6ca40 -r 095d2f074f89ba34bec0c8d96e54a048ce24ff93 OpenCTL/tests/library/TestCompileThreading.h
--- a/OpenCTL/tests/library/TestCompileThreading.h	Thu Jun 07 21:06:28 2012 +1000
+++ b/OpenCTL/tests/library/TestCompileThreading.h	Thu Jun 07 21:10:01 2012 +1000
@@ -19,6 +19,8 @@
 
 #include "GTLTest/Thread.h"
 
+#include <unistd.h>
+
 #ifdef GTLTEST_HAVE_THREAD
 
 class MyThreadTCTLCT : public GTLTest::Thread
diff -r 0cbee16e4e3dc60fcf192035b60f44ae4cd6ca40 -r 095d2f074f89ba34bec0c8d96e54a048ce24ff93 OpenCTL/tests/library/TestRuntimeThreading.h
--- a/OpenCTL/tests/library/TestRuntimeThreading.h	Thu Jun 07 21:06:28 2012 +1000
+++ b/OpenCTL/tests/library/TestRuntimeThreading.h	Thu Jun 07 21:10:01 2012 +1000
@@ -19,6 +19,8 @@
 
 #include "GTLTest/Thread.h"
 
+#include <unistd.h>
+
 #ifdef GTLTEST_HAVE_THREAD
 
 class MyThreadTCTLRT : public GTLTest::Thread
diff -r 0cbee16e4e3dc60fcf192035b60f44ae4cd6ca40 -r 095d2f074f89ba34bec0c8d96e54a048ce24ff93 OpenGTL/GTLCore/tests/TestMemoryManagerThread.h
--- a/OpenGTL/GTLCore/tests/TestMemoryManagerThread.h	Thu Jun 07 21:06:28 2012 +1000
+++ b/OpenGTL/GTLCore/tests/TestMemoryManagerThread.h	Thu Jun 07 21:10:01 2012 +1000
@@ -18,6 +18,7 @@
  */
 
 #include "GTLTest/Thread.h"
+#include <unistd.h>
 
 #ifdef GTLTEST_HAVE_THREAD
 
diff -r 0cbee16e4e3dc60fcf192035b60f44ae4cd6ca40 -r 095d2f074f89ba34bec0c8d96e54a048ce24ff93 OpenShiva/tests/library/TestKernelThreading.h
--- a/OpenShiva/tests/library/TestKernelThreading.h	Thu Jun 07 21:06:28 2012 +1000
+++ b/OpenShiva/tests/library/TestKernelThreading.h	Thu Jun 07 21:10:01 2012 +1000
@@ -21,6 +21,8 @@
 #include "GTLCore/Image.h"
 #include "GTLCore/Type.h"
 
+#include <unistd.h>
+
 #ifdef GTLTEST_HAVE_THREAD
 
 class MyThreadTCTLKT : public GTLTest::Thread
